### DISCUSSION TITLE
本地节点启动后 怎么通过控制台连接节点 让钱包显示余额

### DISCUSSION THREAD

**Discussion by zhaoge5859** [NonceGeek/Move-Camp#157] [Q&A]
已经启动了本地节点并且进入了控制台，对账户增加了余额，也导入了浏览器钱包当中，网络设置为localhost了，但是余额不显示，是不是哪里还缺点什么操作，看了cookbook应该是差一步 控制台连接节点，但是不知道怎么操作，请教各位大佬，谢谢啦

![WechatIMG18](https://user-images.githubusercontent.com/116735858/198866342-ed38cd7e-036b-4d61-82e0-8d1cd5ce53f7.jpeg)
![WechatIMG19](https://user-images.githubusercontent.com/116735858/198866343-952ae762-f3e2-458d-a079-d4876e1c936c.jpeg)
![WechatIMG20](https://user-images.githubusercontent.com/116735858/198866345-83b7ad8b-e1c0-4097-a4b1-78c01be1ff98.jpeg)

**Comment by mason0510** [comment]
节点没连上

**Comment by v1xingyue** [comment]
命令行终端(默认使用ipc文件连接)显示该账户下已经有STC了，那就是已经获取到测试STC了。 
starmask 不显示，应该是 网络连接状况的问题。 
可以按照如下的步骤排查： 

1. 检查自己starcoin node 监听的IP 地址端口 ，可以使用 ss , netstat, lsof 相关命令
2. 如果starcoin 节点在虚拟中，或者 docker 中 需要将端口对外映射出来
3. 按照正确的地址，修改 starmask 连接的节点地址。
